### PR TITLE
added paragraph of text

### PR DESCRIPTION
**Pull request by jennifer-collins**
- Added lorem ipsum text as a paragraph.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 </head>
 <body>
     <h1>What a great website!</h1>
+    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et laudantium, impedit nulla ad aliquid, delectus quibusdam repudiandae labore veritatis esse minima id neque voluptatem sunt, tempora repellendus voluptas! Explicabo nobis maxime veniam eius sequi, delectus odit molestias, ad excepturi deserunt nemo quia fugiat aut inventore animi accusantium nam impedit. Et, culpa a eligendi ad eaque expedita nobis molestias doloremque quam.</p>
```
Comment (seannastewart) [inline index.html]: this looks great but can we remove the last sentence?

**Review by seannastewart — CHANGES_REQUESTED** [review]
Looks good, just a couple things to clean up

File: index.html (hunk 1 above)
Comment (jennifer-collins) [reply index.html]: I have removed the bad sentence.

**Comment by jennifer-collins** [comment]
Made requested changes.

**Review by seannastewart — APPROVED** [review]
all good! go ahead and merge

**Pull request merged**